woocommerce/woocommerce-ios
# Shipping Labels: Update logo for DHL

Fixes #4888 

# Description
This PR updates logo for DHL (previously FedEx's). The logo is square so the aspect ratio is good too.

# Screenshots
| Before | After |
| ----- | ----- |
| <img src="https://user-images.githubusercontent.com/495617/131113026-3fc421be-b3e5-42c5-a7a2-819444ffead4.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/135596576-fc99d17d-6ef4-485c-b530-4077f644ad6b.png" width=320 />

# Testing
1. Make sure your test store has installed and activated WCShip plugin.
2. Select an international order (value around $200 to make sure DHL is available 🤞) > Create Shipping Label.
3. Configure Ship From, Ship To, Package Details, Customs form.
4. On Carriers and Rates screen: notice that the logo for DHL is correct and aspect ratio is good too.

Update release notes:
